tools/test-runner/src/pyunit.rs

```diff
@@ -0,0 +1,63 @@
+use std::io;
+use std::io::{BufRead, BufReader};
+use std::process::Child;
+
+use super::buck_test::{make_command, shell, Test, TestKind, TestSpec};
+
+pub fn validate(spec: TestSpec) -> Vec<Test> {
```

Comment:
Implementation looks pretty good, but I don't really like the name of this function, it doesn't actually do any kind of spec validation afaict, just listing the test cases out. So you can just update the name to something like `list_tests` instead?

Reply:
Done. Now the only function called validate is the top-level one called by main, which actually does some validation.